trimmer_decorator

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
```

Comment:
- [ ] I suggest renaming this file to `trimmer_decorator.rb` because not all text editors can detect that what is in this file is ruby code and will throw an error.